bindings/wasm/examples/src/0_basic/7_revoke_vc.ts

```diff
@@ -0,0 +1,157 @@
+
+    // Generate a random mnemonic for our wallet.
+    const secretManager: MnemonicSecretManager = {
+        mnemonic: Bip39.randomMnemonic(),
+    };
+
+    // Create an identity for the issuer with one verification method `key-1`.
+    let { document: issuerDocument, keypair: keypairIssuer } = await createDid(client, secretManager);
+
+    // Create an identity for the holder, in this case also the subject.
+    const { document: aliceDocument } = await createDid(client, secretManager);
+
+    // Create a new empty revocation bitmap. No credential is revoked yet.
+    const revocationBitmap = new RevocationBitmap();
+
+    // Add the revocation bitmap to the DID Document of the issuer as a service.
+    const service: IotaService = new IotaService({
+        id: issuerDocument.id().join("#my-revocation-service"),
+        type: RevocationBitmap.type(),
+        serviceEndpoint: revocationBitmap.toEndpoint(),
+    });
+    issuerDocument.insertService(service);
+
+    // Resolve the latest output and update it with the given document.
+    let aliasOutput: IAliasOutput = await didClient.updateDidOutput(issuerDocument);
+
+    // Because the size of the DID document increased, we have to increase the allocated storage deposit.
+    // This increases the deposit amount to the new minimum.
+    let rentStructure: IRent = await didClient.getRentStructure();
+    aliasOutput.amount = TransactionHelper.getStorageDeposit(aliasOutput, rentStructure).toString();
+
+    // Publish the document.
+    issuerDocument = await didClient.publishDidOutput(secretManager, aliasOutput);
+
+    // Create a credential subject indicating the degree earned by Alice, linked to their DID.
+    const subject = {
+        id: aliceDocument.id(),
+        name: "Alice",
+        degreeName: "Bachelor of Science and Arts",
+        degreeType: "BachelorDegree",
+        GPA: "4.0",
+    };
+
+    // Create an unsigned `UniversityDegree` credential for Alice.
+    // The issuer also chooses a unique `RevocationBitmap` index to be able to revoke it later.
+    const CREDENTIAL_INDEX = 5;
+    const unsignedVc = new Credential({
+        id: "https://example.edu/credentials/3732",
+        type: "UniversityDegreeCredential",
+        credentialStatus: {
+            id: issuerDocument.id() + "#my-revocation-service",
+            type: RevocationBitmap.type(),
+            revocationBitmapIndex: CREDENTIAL_INDEX,
+        },
+        issuer: issuerDocument.id(),
+        credentialSubject: subject,
+    });
+
+    // Sign Credential.
+    let signedVc = issuerDocument.signCredential(unsignedVc, keypairIssuer.private(), "#key-1", ProofOptions.default());
+    console.log(`Credential JSON > ${JSON.stringify(signedVc, null, 2)}`);
+
+    // ===========================================================================
+    // Revocation of the Verifiable Credential.
+    // ===========================================================================
+
```

Comment:
Perhaps we should also demonstrate that the credential can be verified successfully prior to revocation?